src/shared/components/Posts/components/PostsList/PostsList.tsx

```diff
@@ -0,0 +1,41 @@
+import { FC, useContext } from 'react';
+import { Link } from 'react-router-dom';
+//mui
+import { Card } from '@mui/material';
+//context
+import { PostsContext } from '../../context/PostsContext';
+//components
+import { SinglePost } from '../SinglePost';
+import { Loader } from 'shared/components/Loader';
+//styles
+import styles from './PostsList.module.scss';
+
+const PostsList: FC = ({ children }) => {
+    const { filteredPosts, loading } = useContext(PostsContext);
+
```

Comment:
const noPosts = filteredPosts.length === 0 && !loading and then just put noPosts && in line 16